scripts/globals/abilities/stay.lua

```diff
@@ -21,6 +25,30 @@ end;
 -- onUseAbility
 -----------------------------------
 
-function onUseAbility(player,target,ability)
-    player:petStay();
+function onUseAbility(player,target,ability,action)
+    pet = player:getPet();
```

Comment:
Yes it has to be defined as `local pet` or else it's defined as a global variable. Which I think it stupid on luas part.